lib/qxcli/commands/Compile.js

```diff
@@ -146,41 +143,38 @@ qx.Class.define("qxcli.commands.Compile", {
             describe: "Deletes the target dir before compile",
             type: "boolean"
           }
-          
         },
         handler   : function(argv){
           return new qxcli.commands.Compile(argv)
             .process()
             .catch((e) => {
               console.error(e.stack || e.message);
-              process.exit(1);
             });
         }
       };
     }
-    
   },
-  
+
   members: {
     __gauge: null,
-    
+
     /*
      * @Override
      */
     process: async function() {
-      if (this.argv["machine-readable"])
+      if (this.argv["machine-readable"]) {
         qxcompiler.Console.getInstance().setMachineReadable(true);
-      else if (this.argv["feedback"]) {
+      } else if (this.argv["feedback"]) {
         this.__gauge = new Gauge();
         this.__gauge.show("Compiling", 0);
         const TYPES = {
-          "error": "ERROR",
-          "warning": "Warning"  
+          error: "ERROR",
+          warning: "Warning"
         };
-        qxcompiler.Console.getInstance().setWriter((str, msgId) => { 
+        qxcompiler.Console.getInstance().setWriter((str, msgId) => {
           msgId = qxcompiler.Console.MESSAGE_IDS[msgId];
           if (msgId.type !== "message")
-            console.log('\n' + TYPES[msgId.type] + ": " + str);
```

Comment:
why is the `\n` removed?  it's important to prevent the gas gauge from overwriting the error message

Reply:
It's merge Problem..